frontend/src/app/components/MobileDialog/index.js

```diff
@@ -0,0 +1,284 @@
+// @flow

```

Comment:
Looks like you meant to move it into a shared component, but [there's still a copy of `MobileDialog.js` in the `app/containers/ExperimentContainer`](https://github.com/mozilla/testpilot/blob/5e01e25f701cb6fe1ff9c2b78229b89faf8df3cf/frontend/src/app/containers/ExperimentPage/MobileDialog.js).

Reply:
Oh whoops, removing now, thank you